app/javascript/controllers/search_filters_controller.js

```
@@ -219,9 +259,111 @@ export default class extends Controller {
+            this.buttonTarget.classList.remove("filters-active");
+            this.removeFilterBadge();
+        }
+    }
+
+    /**
+     * Check if there are any active filters
+     * @returns {boolean}
+     */
+    hasActiveFilters() {
+        if (
+            this.currentResourceValue === "Projects" &&
```

ESLint

[error] '&&' should be placed at the beginning of the line.

(operator-linebreak)